app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java

```diff
@@ -67,9 +67,8 @@
         if (session.room != null) {
             this.roomName = session.room.name;
         }
-        if (session.lang != null) {
-            this.languageId = session.lang.toUpperCase();
-        }
+       
+        this.languageId = session.lang != null ? session.lang.toUpperCase() : Session.LANG_EN_ID;
```

Comment:
Thanks reviewing!
I fix it. when `session.lang` is null,  make `txt_language`  hide.